health-services/individual/src/main/resources/db/migration/main/V20230302155500__alter_individual_address_ddl.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE INDIVIDUAL ADD COLUMN individualId character varying(64) UNIQUE;
+ALTER TABLE INDIVIDUAL ADD COLUMN relationship character varying(100);
+
+ALTER TABLE ADDRESS ADD COLUMN wardCode character varying(256);
```

Comment:
Put an if exists check here, for the address table column alter.

Reply:
This is not changed because the sql is deployed to works dev and any modification on this will file will cause flyway checksum failure